lib/ansible/modules/uri.py

```diff
@@ -443,11 +443,10 @@
 import os
 import re
 import shutil
-import sys
 import tempfile
 
 from ansible.module_utils.basic import AnsibleModule, sanitize_keys
-from ansible.module_utils.six import PY2, PY3, binary_type, iteritems, string_types
+from ansible.module_utils.six import binary_type, iteritems, string_types
```

Comment:
`binary_type` is always aliased to `bytes` under Python 3. There are two places where it's used and can be replaced.
https://six.rtfd.io/#six.binary_type

Similarly, `string_types` is always `str`: https://six.rtfd.io/#six.string_types. It is used in 4 places in this module. We can replace it too.

`iteritems(dict())` is used in a single place and is equivalent to `dict().items()`: https://six.rtfd.io/#six.iteritems.

So maybe let's get rid of the entire `six` import?
```suggestion
```

Reply:
I agree with the changes however my intent for this PR was to basically remove unreachable code guarded under `PY2`/`not PY3`. For changes like `string_types` -> `str` I would prefer them being done separately (either in individual PRs when changing related code or, if need be, in a bulk in one PR).